tests/repl.sh

```diff
@@ -105,18 +105,13 @@ testReplResponseNoRegex '
 testReplResponse '
 drvPath
 ' '".*-simple.drv"' \
-$testDir/simple.nix
```

Comment:
The test suite always enables `flakes`, to my discontent. Fixing that would be a lot of work for little effect, so this is the way to go.